docs/website/developers/committers/release-process/publish.md

```diff
@@ -69,12 +74,15 @@ link in the top right (the credentials are the same as your Git and Jenkins cred
 Repositories" page, and tick the repository with the name starting `orgapachebrooklyn`. Click the **Release** button.
 Provide a description which includes the version, e.g. `Apache Brooklyn 0.7.0-incubating`.
 
+Note the staging repository may have a name like "080 RC2". You can double-check under the "content" that brooklyn-dist
```

Comment:
i don't think we need the added text as the RM will have given the repo name.  it won't look like `080-RC2` unless it was me :) .